core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1942,6 +1950,15 @@ package object config {
       .timeConf(TimeUnit.SECONDS)
       .createOptional
 
+  private[spark] val EXECUTOR_DECOMMISSION_CLEANUP_INTERVAL =
+    ConfigBuilder("spark.executor.decommission.cleanupInterval")
```

Comment:
Shall we rename it to `*.decommission.forceKillTimeout`? `Interval` sounds more like a periodical task while it actually happens once.

Reply:
Yeah sure that sounds like a reasonable change.